app/controllers/ems_common.rb

```
@@ -548,6 +549,26 @@ def process_emss(emss, task)
          :product => I18n.t('product.name'),
          :model   => ui_lookup(:table => table_name),
          :models  => ui_lookup(:tables => table_name)}) if @flash_array.nil?
+    elsif task == "disable_ems"
```

please raise an error emss if there is more then one ems